### PR TITLE
Ensure methods accept an empty dictionary.

### PR DESCRIPTION
**Pull request by Grahams-Uncle**
Some methods don't work with empty dictionary

### CODE REVIEW

**Review by BirkhoffG — CHANGES_REQUESTED** [review]
You should ensure those methods accept an empty dictionary. 

You can emulate VanillaCF's implementation: https://github.com/BirkhoffG/ReLax/blob/201d0776f6a7ac536b6028b2b72912a0d8328a90/relax/methods/vanilla.py#L77-L79

File: relax/methods/clue.py [hunk 1]
```diff
@@ -348,7 +348,7 @@ class CLUE(BaseCFModule, BaseParametricCFModule):
     name: str = 'CLUE'
 
     def __init__(self, m_config: Dict | CLUEConfigs = None):
-        if m_config is None: m_config = CLUEConfigs()
+        if m_config is None or not bool(m_config): m_config = CLUEConfigs()
```
Comment (BirkhoffG) [inline relax/methods/clue.py:351]: You should write:

```python
if m_config is None:
        m_config = CLUEConfigs()
self.m_config = validate_configs(m_config, CLUEConfigs)
```

File: relax/methods/vaecf.py [hunk 2]
```diff
@@ -259,7 +259,7 @@ class VAECF(BaseCFModule, BaseParametricCFModule):
     name: str = 'VAECF'
 
     def __init__(self, m_config: Dict | VAECFConfigs = None):
-        if m_config is None:
+        if m_config is None or not bool(m_config):
```
Comment (BirkhoffG) [inline relax/methods/vaecf.py:262]: Same here

**Review by BirkhoffG — CHANGES_REQUESTED** [review]

File: relax/methods/clue.py (hunk 1 above)
Comment (Grahams-Uncle) [reply relax/methods/clue.py:351]: validate_configs doesn't capture the empty dict case, and raises error:
  File "/Users/chuck/PycharmProjects/temp/ReLax/relax/methods/vaecf.py", line 266, in __init__
    self.module = VAECFModule(m_config.dict())
AttributeError: 'dict' object has no attribute 'dict'

File: relax/methods/vaecf.py (hunk 2 above)
Comment (Grahams-Uncle) [reply relax/methods/vaecf.py:262]: same error

**Comment by BirkhoffG** [comment]
In the future, try to write commit messages which summarize what this commit does. Don't write something like: "Address comments". Write "Fix methods to accept empty dictionary via `validate_configs`".

**Review by BirkhoffG — APPROVED** [review]

**Pull request merged**